src/views/ErrorCodes.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <v-alert type="error" color="pink darken-4">
+    <v-row align="center">
+      <v-col class="grow">
+        <div v-html="errorMessage"></div>
+      </v-col>
+      <v-col class="shrink"> </v-col>
+    </v-row>
+  </v-alert>
+</template>
```

Comment:
How can I go back to the main page once I do what the error message tells me to do? On "about" pages I have the "Back" button, but I can't see any analogue here.